src/DataTable/index.tsx

```diff
@@ -208,20 +208,20 @@ export const DataTable = <T extends Record<string, unknown>>(
     setData(data)
   }, [data])
 
-  const handleFetchDataDebounced = useAsyncDebounce(handleFetchData, 200)
-
-  // If an handleFetchData handler is passed, use it to pull new data on page change
-  useEffect(() => {
-    if (
-      handleFetchData &&
-      (
-        (pageSize !== prevPageProps?.pageSize && pageSize > 0) ||
-        pageIndex !== prevPageProps?.pageIndex
-      )
-    ) {
-      handleFetchDataDebounced({ pageIndex, pageSize, sortBy })
-    }
-  }, [useAsyncDebounce, handleFetchData, pageIndex, pageSize, sortBy])
+  let handleFetchDataDebounced: (args: HandleFetchDataArgs<T>) => Promise<void>
```

Comment:
Is this used elsewhere? We may want to just create this function inside of the if block instead of redefining it.

Reply:
Yes, this is used on line 316.

Reply:
> Is this used elsewhere? We may want to just create this function inside of the if block instead of redefining it.

Yes, this is used on line 316.